shorts_pipeline/modules/voice_synthesizer.py

```
@@ -0,0 +1,56 @@
+    def _local_tts(self, text: str, dst: Path):
+        model = "tts_models/multilingual/multi-dataset/xtts_v2"
+        tts = TTS(model_name=model, gpu=True)
+        tts.tts_to_file(text=text, file_path=str(dst))
+
+    async def _process_script(self, session, script: Script):
+        text = f"{script.hook}. {script.content}. {script.retention_hook}."
+        audio_path = self.out_dir / f"script_{script.id}.mp3"
+        try:
+            async with self.sem:
+                await self._elevenlabs(text, audio_path)
+        except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)